draft-ietf-webtrans-http3.md

```diff
@@ -240,56 +241,53 @@ type, followed by the session ID, encoded as a variable-length integer, followed
-  |                           0x54 (i)                          ...
-  +-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+
-  |                        Session ID (i)                       ...
-  +-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+
-  |                         Stream Body                         ...
-  +-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+
+Stream Type (i) = 0x54,
+Session ID (i),
+Stream Body (..),
 ~~~~~~~~~~
 {: #fig-unidi title="Unidirectional WebTransport stream format"}
 
 ## Bidirectional Streams
 
-WebTransport endpoints can initiate bidirectional streams by opening an HTTP/3
-bidirectional stream and then immediately sending a special signal value 0x41,
-followed by the associated session ID, both encoded as a variable-length
-integer; the rest of the stream is the application payload of the WebTransport
-stream ({{fig-bidi-client}}).
+All client-initiated bidirectional streams are reserved by HTTP/3 as request
+streams, which are a sequence of HTTP/3 frames with a variety of rules; see
+{{Sections 4.1 and 6.1 of HTTP3}}.
+
+WebTransport extends HTTP/3 to allow clients to declare and use alternative
+request stream rules.  Once a client receives settings indicating WebTransport
+support ({{establishing}}), it can send a special signal value, encoded as
+a variable-length integer, as the first bytes of the stream in order to indicate
+how the remaining bytes on the stream are used.
+
+WebTransport extends HTTP/3 by defining rules for all server-initiated
+bidirectional streams.  Once a server receives settings indicating WebTransport
+support ({{establishing}}), it can open a bidirectional stream and SHALL send a
+special signal value, encoded as a variable-length integer, as the first bytes
+of the stream in order to indicate how the remaining bytes on the stream are
+used.
+
+The signal value, 0x41, is used by clients and servers to open a bidirectional
+WebTransport stream.  Following this is the associated session ID, encoded as a
+variable-length integer; the rest of the stream is the application payload of
+the WebTransport stream ({{fig-bidi-client}}).
 
 ~~~~~~~~~~ drawing
-  0                   1                   2                   3
-  0 1 2 3 4 5 6 7 8 9 0 1 2 3 4 5 6 7 8 9 0 1 2 3 4 5 6 7 8 9 0 1
-  +-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+
-  |                           0x41 (i)                          ...
-  +-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+
-  |                        Session ID (i)                       ...
-  +-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+
-  |                         Stream Body                         ...
-  +-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+
+Signal Value (i) = 0x41,
+Session ID (i),
+Stream Body (..),
 ~~~~~~~~~~
 {: #fig-bidi-client title="Bidirectional WebTransport stream header"}
 
```

Comment:
Shouldn't there be some curly braces, as in:
~~~
Bidirectional Stream {
    Signal Value (i) = 0x41,
    Session ID (i),
    Stream Body (..)
}
~~~

Reply:
Indeed (both here and in the diagram above).